src/App/Pages/Send/SendAddEditPageViewModel.cs

```diff
@@ -96,6 +100,16 @@ public SendAddEditPageViewModel()
         public bool ShareOnSave { get; set; }
         public bool DisableHideEmailControl { get; set; }
         public bool IsAddFromShare { get; set; }
+        public bool CopyInsteadOfShareAfterSaving
```

Comment:
Move this down so it's grouped with the other properties with custom get/set logic like `SendEnabled`, etc.)